e2e-pw/src/oss/specs/3d/pcd-only-dataset.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { test as base, expect } from "src/oss/fixtures";
+import { GridPom } from "src/oss/poms/grid";
+import { ModalPom } from "src/oss/poms/modal";
+import { getUniqueDatasetNameWithPrefix } from "src/oss/utils";
+
+const datasetName = getUniqueDatasetNameWithPrefix(
+  "pcd-orthographic-projection"
+);
+const normalPcd = `/tmp/test-pcd1-${datasetName}.pcd`;
+const pcdWithNaN = `/tmp/test-pcd2-${datasetName}.pcd`;
+
+const test = base.extend<{ grid: GridPom; modal: ModalPom }>({
+  grid: async ({ page }, use) => {
+    await use(new GridPom(page));
+  },
+  modal: async ({ page }, use) => {
+    await use(new ModalPom(page));
+  },
+});
+
+test.describe("orthographic projections", () => {
+  test.beforeAll(async ({ fiftyoneLoader, mediaFactory }) => {
+    mediaFactory.createPcd({
+      outputPath: normalPcd,
+      shape: "cube",
+      numPoints: 100,
+    });
+    mediaFactory.createPcd({
+      outputPath: pcdWithNaN,
+      shape: "cube",
+      numPoints: 100,
+      imputeNaN: {
+        indices: [
+          [0, 0],
+          [1, 1],
+          [2, 2],
+        ],
+      },
+    });
+
+    await fiftyoneLoader.executePythonCode(
+      `
+      import fiftyone as fo
+      import fiftyone.utils.utils3d as fou3d
+
+      dataset = fo.Dataset("${datasetName}")
+      dataset.persistent = True
+
+      sample1 = fo.Sample(filepath="${normalPcd}")
+      sample2 = fo.Sample(filepath="${pcdWithNaN}")
+      dataset.add_samples([sample1, sample2])
+
+      fou3d.compute_orthographic_projection_images(dataset, (-1, 64), "/tmp/ortho") 
+      `
+    );
+  });
+
+  test.beforeEach(async ({ page, fiftyoneLoader }) => {
+    await fiftyoneLoader.waitUntilGridVisible(page, datasetName);
+  });
+
+  test("orthographic projections are rendered correctly", async ({
+    modal,
+    grid,
+  }) => {
+    await expect(grid.firstFlashlightSection).toHaveScreenshot(
+      "orthographic-projection-grid-cuboids.png",
+      {
+        animations: "allow",
+        mask: [modal.locator.getByTestId("looker3d-action-bar")],
```

Comment:
Perhaps there is a way to avoid the mask by ensuring Looker3D is blurred (out of focus)